src/backends/github/API.js

```diff
@@ -386,7 +386,7 @@ export default class API {
     return this.deleteRef("heads", branchName);
   }
 
-  createPR(title, head, base = "master") {
+  createPR(title, head, base = this.branch) {
```

Comment:
Wondering if that `Super expression` error you saw was related to this use of `this`.